### PR TITLE
updating Meetup Pro guidelines (and more)

### LINKED ISSUE #101: remove mentions to current_chapters.csv

### LINKED ISSUE #103: remove mentions to the starter kit repo

### LINKED ISSUE #104: change name of global team members to global team roles

### LINKED ISSUE #105: acknowledgements global team members
Link to global team members here is broken: https://guide.rladies.org/acknowledgements/ replace https://rladies.org/about-us/team/ with https://rladies.org/about-us/global-team/

Also, should we continue to keep this named list updated, or just reference the updated link as that has all current and former global team members?

### PR DESCRIPTION
**Pull request by rivaquiroga**
Hello hello!

- I updated the Meetup Guidelines for both Global Team members and chapter organizers. They now reflect the current workflow. I also edited the format of the message templates, so the Meetup Team can copy paste the from the Guide to Slack, and from the Guide to Meetup without the need to edit them afterwards. I also removed the Twitter account from the templates and added the new Mastodon one 🌈 . 
- I removed almost all mentions to `current_chapters.csv`, the starter kit repo, and to using the info account for new chapters. The ones that I didn't remove were the ones on /coordination/onboarding/. The reason is that that section requires a major update, but that can wait another week or two (we are not currently onboarding someone to that team). In all the sections intended for organizers, the update was made. (Closes #101. Closes #103)
- I removed mentions of (former) Global team members when talking about tasks and workflows. I changed them for role names or actions (e.g. "inform Florencia D’Andrea" > "inform the Mentoring Team"). (Closes #104)
- I removed a mention to using Twitter on the About page. I changed it to “social media”.
- I updated some broken links. (Closes #105)
- As discussed, I changed the statemente about Etiquette for Slack: channels are the primary form of communication

### CODE REVIEW

**Comment by rivaquiroga** [comment]
This is still a draft BTW (I changd the status because I wanted to see the preview)

File: content/acknowledgements.en.md [hunk 1]
```diff
@@ -28,7 +28,9 @@ _Grácias a todas las personas que contribuyeron al contenido de esta guía, aqu
   Page Piccinini, 
   Patricia A. Loto,
```
Comment (yabellini) [inline content/acknowledgements.en.md:29]:
```suggestion
  Paola Corrales, 
  Patricia A. Loto, 
```

File: content/acknowledgements.en.md [hunk 2]
```diff
@@ -52,7 +54,6 @@ _Grácias a todas las personas que contribuyeron al contenido de esta guía, aqu
   Liili Abuladze,
```
Comment (yabellini) [inline content/acknowledgements.en.md:54]: We should remove Pao Corrales from this list because she have to be on the Global Team list.

**Review by yabellini — COMMENTED** [review]
This looks very nice Riva! Thanks so much for all this work.  I have a small change request.

File: content/acknowledgements.en.md (hunk 2 above)
Comment (rivaquiroga) [reply content/acknowledgements.en.md:55]: Done!

**Pull request merged**